mlflow/tracking/fluent.py

```diff
@@ -69,9 +70,29 @@
     import numpy
     import PIL
 
-_active_run_stack = []
-_active_experiment_id = None
-_last_active_run_id = None
+
```

Comment:
We should add a note for developer here saying:

`fluent tracking APIs` must be implemented in thread-safe way, so that if you want to add new global variable that is used by fluent tracking API implementation, please wraps it with `DefaultSetContextVar`